flow/packet-headers/ipv6_routing.yaml

```diff
@@ -0,0 +1,397 @@
+          x-field-uid: 8
+    Flow.Ipv6SegmentRouting.Flags:
+      description: An 8-bit field containing flags. While RFC 8754 reserves all
+        bits as unused, earlier drafts defined specific flags for behavior such
+        as OAM, HMAC, and FRR protection.
+      type: object
+      properties:
+        u1_flag:
+          x-field-pattern:
+            description: Unused bit. Should be set to 0.
+            format: integer
+            length: 1
+            default: 0
+            features:
+              - count
+          x-field-uid: 1
+        p_flag:
+          x-field-pattern:
+            description: Protected Flag. Indicates if the segment is protected
+              by a Fast Re-Route (FRR) mechanism.
+            format: integer
+            length: 1
+            default: 0
+            x-constants:
+              not_protected: 0
+              protected: 1
+            features:
+              - count
+          x-field-uid: 2
+        o_flag:
+          x-field-pattern:
+            description: OAM Flag. Indicates if the packet is an Operations, Administration,
+              and Maintenance (OAM) packet.
+            format: integer
+            length: 1
+            default: 0
+            x-constants:
+              not_oam: 0
+              oam: 1
+            features:
+              - count
+          x-field-uid: 3
+        a_flag:
+          x-field-pattern:
+            description: Alert Flag. Indicates the presence of important TLVs
+              that must be inspected.
+            format: integer
+            length: 1
+            default: 0
+            x-constants:
+              no_alert: 0
+              alert: 1
+            features:
+              - count
+          x-field-uid: 4
+        h_flag:
+          x-field-pattern:
+            description: HMAC Flag. Indicates the presence of the HMAC TLV for
+              verification.
+            format: integer
+            length: 1
+            default: 0
+            x-constants:
+              no_hmac: 0
+              hmac_present: 1
+            features:
+              - count
+          x-field-uid: 5
+        u2_flag:
+          x-field-pattern:
+            description: Unused 3 bits. Should be set to 0.
+            format: integer
+            length: 3
+            default: 0
+            features:
+              - count
+          x-field-uid: 6
+    Flow.Ipv6SegmentRouting.TlvPad:
+      description: Padding TLV used when more than one byte of padding is required
+        to align subsequent TLVs or pad the header to an 8-octet boundary.
+      type: object
+      properties:
+        padding:
+          description: Variable-length padding bytes. Must be set to 0 on transmission.
+          type: string
+          format: hex
+          default: '00'
+          x-field-uid: 1
+    Flow.Ipv6SegmentRouting.TlvIngressNode:
+      description: A TLV that may be used to carry the IPv6 address of the ingress
```

Comment:
Reference to RFC esp. to Type value and calling it out specifically would be useful for all the TLVs in the model here,